README.md

```diff
@@ -32,10 +32,10 @@ The second sample AzureSearchDotNetSample is a more detailed example that demons
 
 ## More information
 
-For more details on these samples, please refer to these articles on Azure.com:
+For more details on these samples, please refer to these articles:
 
-  - [How to use Azure Search from a .NET Application](https://azure.microsoft.com/en-us/documentation/articles/search-howto-dotnet-sdk/).
-  - [Get started with your first Azure Search application in .NET](https://azure.microsoft.com/en-us/documentation/articles/search-get-started-dotnet/).
+  - [How to use Azure Search from a .NET Application](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).
+  - [Get started with your first Azure Search application in .NET](https://docs.microsoft.com/azure/search/search-howto-dotnet-sdk).

```

Comment:
It turns out the second ACOM article was deleted, so both links re-directed to the same article. I've just removed the second link.